app/helpers/application_helper.rb

```diff
@@ -381,8 +381,8 @@ def db_to_controller(db, action = "show")
       controller = "infra_networking"
       action = @lastaction
     when "MiqReportResult"
-      controller = "report"
-      action = "show_saved"
+      controller = params[:controller]
```

Comment:
This is dangerous, we should not rely on something dynamic like this if possible.